src/core/js/views/drawerView.js

```diff
@@ -118,7 +118,7 @@ define([
         },
 
         showDrawer: function(emptyDrawer) {
-            this.$el.removeClass('display-none');
+            this.$el.removeClass('display-none').removeAttr('aria-hidden');
```

Comment:
With accessibility becoming a default I think we should use aria-attributes to control an elements visibility instead of maintaining classes.
```
.drawer[aria-hidden="true"] {
  display: none;
}
```

Reply:
aria-hidden != display: none

there are quite a number of places where you'd want aria-hidden by display:block
i'm not sure this association is a good way to think about it.

in this instance it would work, i can change it if you want?

Reply:
I did not state that aria-hidden should always be set to display none.
Instead of adding a class to handle the state based styling we could use the attribute which is set anyway. This would make it also more descriptive and make sure that we use the appropriate aria labels.

Reply:
The aria spec is intentionally separate from css, html and ecmascript and is specifically for screen readers only, using it for styling seems a counter intuitive to me, like using a data attribute for styling instead of a class. It's the separation of concerns I was trying to speak to.

Reply:
Fine for me ... just thought we can reuse logic ...

Reply:
reference article: https://medium.com/@heyoka/use-aria-attributes-for-javascript-state-setting-styling-efa87457a613